README.md

```diff
@@ -48,3 +48,11 @@ You can also search for relevant files and sub-datasets as follows:
 ```console
 datalad search T1
 ```
+
+## Contributing
+
+This repository has multiple git submodules pointing at dataset instances available in the [conpdatasets](https://github.com/conpdatasets) repository.
+Therefore, to contribute to such datasets it is required that collaborators have push access to both repositories.
+
+In addition, it is important to keep in mind that the conpdatasets repository is a fork of the [OpenNeuroDatasets](https://github.com/OpenNeuroDatasets/ds001634) project. 
```

Comment:
Not true for all of the datasets.
Prevent-AD is not on OpenNeuro.
Same for Multicenter-phantom.

Reply:
Hi @PapillonMcGill, thank you for your comment, I will close the PR for the moment, as this information as you say may not be correct.